util/is-inverse-benchmark-record.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {{}} record
+ * @return {boolean}
+ */
+var isInverseBenchmarkRecord = function isInverseBenchmarkRecord(record) {
```

Comment:
Would defer to @marimiyachi but the pattern in this repo for function declarations to omit the var declaration, e.g.:

```javascript
function isInverseBenchmarkRecord(record) {
  ...
}
```